webserver/grpc/src/main/java/io/helidon/webserver/grpc/GrpcProtocolHandler.java

```diff
@@ -317,6 +315,7 @@ public void sendHeaders(Metadata headers) {
                                           streamId,
                                           HeaderFlags.create(END_OF_HEADERS),
                                           flowControl.outbound());
+                headersSent = true;
```

Comment:
Question: Do we know for sure that `sendHeaders` will be invoked at most once, so there's no need to check `headersSent` at the beginning of the method to avoid sending them twice?

Reply:
Yes, here is the Javadoc for that method:

```
Send response header metadata prior to sending a response message. This method may only be called once and cannot be called after calls to sendMessage or close.
```